predictive-search-box/react-instantsearch/src/lib/PredictiveSearchBox.js

```diff
@@ -109,6 +109,8 @@ class PredictiveSearchBox extends Component {
                     onChange={event =>
                         this.refineSuggestionsAndSearch(event.target.value)
                     }
+                    onBlur={() => this.setState({ ...this.state, focus: false })}
```

Comment:
setState already does a shallow merge, you can change this to just `focus: false`